base/special/exp.jl

```diff
@@ -0,0 +1,137 @@
+# Based on FreeBSD lib/msun/src/e_exp.c
+# ====================================================
+# Copyright (C) 2004 by Sun Microsystems, Inc. All rights reserved. Permission
```

Comment:
```julia 
# ====================================================
# Copyright (C) 2004 by Sun Microsystems, Inc. All rights reserved. Permission
# to use, copy, modify, and distribute this software is freely granted,
# provided that this notice is preserved.
# ====================================================
```

do I even need to include this? We are distributing the original file.  

Is it necessary to add a line here  https://github.com/JuliaLang/julia/blob/master/contrib/add_license_to_files.jl#L39 to skip this? 
I'm not sure. Parts of this include ported and derived code?